Source/CarthageKit/FrameworkExtensions.swift

```diff
@@ -273,7 +274,9 @@ extension URL {
 
 extension FileManager: ReactiveExtensionsProvider {
 	@available(*, deprecated, message: "Use reactive.enumerator instead")
-	public func carthage_enumerator(at url: URL, includingPropertiesForKeys keys: [URLResourceKey]? = nil, options: FileManager.DirectoryEnumerationOptions = [], catchErrors: Bool = false) -> SignalProducer<(FileManager.DirectoryEnumerator, URL), CarthageError> {
+	public func carthage_enumerator(at url: URL, includingPropertiesForKeys keys: [URLResourceKey]? = nil,
+	                                options: FileManager.DirectoryEnumerationOptions = [],
+	                                catchErrors: Bool = false) -> SignalProducer<(FileManager.DirectoryEnumerator, URL), CarthageError> {
```

Comment:
Spaces here are an issue for me.

I'd be in favor of every argument tab-preceded on a new line as has been done other places.